app/prompts.js

```diff
@@ -151,15 +151,34 @@ module.exports = function(advanced, defaults) {
 			name: 'themeType',
 			default: defaults.themeType || 'git',
 			validate: function(value) {
-				if (value != '' && /^(?:git|tar)$/.test(value)) {
+				if (value != '' && /^(?:git|tar)$/i.test(value)) {
 					return true;
 				}
 				return false;
 			},
+			filter: function(value) {
+				return value.toLowerCase();
+			},
 			when: function(res) {
 				return !!res.installTheme;
 			}
 		}, {
+			message: 'Task runner (grunt/gulp)',
+			name: 'themeTaskRunner',
+			default: defaults.themeTaskRunner || 'grunt',
+			validate: function(value) {
+				if (value != '' && /^(?:grunt|gulp)$/i.test(value)) {
+					return true;
+				}
+				return false;
+			},
+			filter: function(value) {
+				return value.toLowerCase();
```

Comment:
Adding the filter here pretty much makes the case insensitive regex unnecessary.  But I can fix this once I merge it in.